dat/CMakeLists.txt

```diff
@@ -1,4 +1,10 @@
+# TODO: This creates lots of files in ${NLE_DAT}, i.e. in the source dir. We
+# should create files in the binary dir only. NetHack's utils require their
+# arguments to be in the same local directory, so this requires copying files
+# from ${NLE_DAT} to ${NLE_DAT_GEN}
```

Comment:
[draft comment -- putting this down as I read the rest of the PR]
This comment confuses me. The way I setup the build process is that there are temporary steps where the nethack src dirs are dirty (because of the various generated data files and headers), but they _should_ get removed or moved into the build dir as soon as it is possible.

This is not ideal, however it's not easy to fix without modifying makedefs, since the binary hardcodes paths wrt. the source dir location, not `$cwd` (it is however fairly trivial to fix it if we are OK with modifying `makedefs` and such).

Reply:
I see, with this PR `*.lev` files are generated in `dat/`.